<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="SpeedGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Speed</string>
    </property>
    <widget class="QDial" name="SpeedDial">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>120</width>
       <height>130</height>
      </rect>
     </property>
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>240</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
     <property name="notchTarget">
      <double>2.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="SpeedBox">
     <property name="geometry">
      <rect>
       <x>55</x>
       <y>160</y>
       <width>40</width>
       <height>25</height>
      </rect>
     </property>
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>240</number>
     </property>
     <property name="value">
      <number>60</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="KeyGroup">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>150</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Key</string>
    </property>
    <widget class="QSlider" name="KeySlide">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>20</width>
       <height>175</height>
      </rect>
     </property>
     <property name="maximum">
      <number>11</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="KeyLables">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>16</y>
       <width>20</width>
       <height>175</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="title">
      <string notr="true"/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QLabel" name="D">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>135</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>D</string>
      </property>
     </widget>
     <widget class="QLabel" name="Eb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Eb</string>
      </property>
     </widget>
     <widget class="QLabel" name="Gb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>75</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Gb</string>
      </property>
     </widget>
     <widget class="QLabel" name="F">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>F</string>
      </property>
     </widget>
     <widget class="QLabel" name="G">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>G</string>
      </property>
     </widget>
     <widget class="QLabel" name="Ab">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>45</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Ab</string>
      </property>
     </widget>
     <widget class="QLabel" name="E">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>105</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>E</string>
      </property>
     </widget>
     <widget class="QLabel" name="A">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>A</string>
      </property>
     </widget>
     <widget class="QLabel" name="C">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>165</y>
        <width>18</width>
        <height>9</height>
       </rect>
      </property>
      <property name="text">
       <string>C</string>
      </property>
     </widget>
     <widget class="QLabel" name="Db">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Db</string>
      </property>
     </widget>
     <widget class="QLabel" name="Bb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>15</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>Bb</string>
      </property>
     </widget>
     <widget class="QLabel" name="B">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>18</width>
        <height>10</height>
       </rect>
      </property>
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="KeyDrop">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>90</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Db</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Eb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ab</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bb</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FormGroup">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>150</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Form Data</string>
    </property>
    <widget class="QSpinBox" name="SectionLengthBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Section&lt;br/&gt;Length&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Repeats</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>100</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lead&lt;br/&gt;Instrument&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>62</x>
       <y>140</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Chord&lt;br/&gt;Instrument&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="RepeatsBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="LeadBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="value">
      <number>66</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="ChordBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>128</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TimeGroup">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>150</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Time Signature</string>
    </property>
    <widget class="QSpinBox" name="TimeBox">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>90</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
     <property name="value">
      <number>4</number>
     </property>
    </widget>
    <widget class="QLabel" name="Time">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>/4</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ButtonGroup">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="StartBut">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="StopBut">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>0</width>
       <height>0</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>210</y>
      <width>191</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="Output">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>191</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>21</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Key</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>0</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Measures</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>21</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>BPM</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>21</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
